src/main/java/net/sydokiddo/auditory/mixin/items/WeaponSwingMixin.java

```diff
@@ -25,110 +22,93 @@
 @Mixin(LivingEntity.class)
 public abstract class WeaponSwingMixin extends Entity {
 
-    @Shadow public boolean swinging;
-
-    @Shadow public int swingTime;
-
-    @Shadow protected abstract int getCurrentSwingDuration();
-
-    @Shadow public InteractionHand swingingArm;
-
     @Shadow public abstract ItemStack getItemInHand(InteractionHand interactionHand);
 
     public WeaponSwingMixin(EntityType<?> entityType, Level level) {
         super(entityType, level);
     }
 
     @Unique
-    private static void playSwordSounds(Entity entity) {
-        entity.level.playSound(null, entity.getX(), entity.getY(), entity.getZ(), ModSoundEvents.ITEM_SWORD_SWING, SoundSource.PLAYERS, 0.1F, 0.8f + entity.level.random.nextFloat() * 0.4F);
+    private void auditory_playSwordSounds() {
```

Comment:
Methods added by mixins should be prefixed with their mod's ID and annotated with `@Unique`.
`@Unique` only prevents conflicts with methods in the target class, so the mod ID prefix is necessary to avoid potential conflicts with other mods.